modules/acs-architecture-overview.adoc

```diff
@@ -9,15 +9,20 @@
 
 [NOTE]
 ====
-The architecture is slightly different when you install {product-title-short} on Kubernetes and in {ocp}.
+The architecture is slightly different when you install {product-title-short} for Kubernetes and {ocp} systems.
 However, the underlying components and the interactions between them remain the same.
 ====
 
-.{product-title-short} architecture for Kubernetes
+.{product-title-short} architecture
 
-The following graphic shows the architecture with the StackRox Scanner. For version 4.4, Scanner V4 is available. Installation of Scanner V4 is optional, but provides additional benefits.
+The following graphic shows the architecture with the StackRox and Scanner V4 components. Installation of Scanner V4 is optional, but provides additional benefits.
```

Comment:
Didn't catch this before, should probably add `Scanner` after `StackRox` to match the UI:

```suggestion
The following graphic shows the architecture with the StackRox Scanner and Scanner V4 components. Installation of Scanner V4 is optional, but provides additional benefits.
```

![image](https://github.com/user-attachments/assets/442e3303-f6cb-4691-8e47-b3d68ecdc013)